sources/commands/Cache.ts

```diff
@@ -0,0 +1,22 @@
+import {Command}          from 'clipanion';
+import fs                 from 'fs';
+
+import {getInstallFolder} from '../folderUtils';
+import type {Context}     from '../main';
+
+export class CacheCommand extends Command<Context> {
+  static paths = [
+    [`cache`, `clean`],
```

Comment:
nit: Even for me it's never clear whether the command is "clean" or "clear". I think it'd make sense to support both, wdyt? 🤔

```suggestion
    [`cache`, `clean`],
    [`cache`, `clear`],
```

Reply:
Sure that doesn't cost us much